Hymns/Settings/FeedbackView.swift

```diff
@@ -0,0 +1,26 @@
+import SwiftUI
+import MessageUI
+
+struct FeedbackView: View {
+
+    @State var result: Result<MFMailComposeResult, Error>?
```

Comment:
What do you do with this result?

Would it be possible to show like a simple toast message telling them that the email succeeded or failed?

We can do this as a follow-up or create an issue for it, since it look like creating a toast message is non-trivial

Reply:
found some pointers here: https://stackoverflow.com/questions/56550135/swiftui-global-overlay-that-can-be-triggered-from-any-view

Reply:
Let's do it as a follow up lol.. I'll log an issue. And the result is used between the UIViewRepresentable and Coordinator as a binding variable passed to both of them.